### DISCUSSION TITLE
Datasette Weekly - a new newsletter

### DISCUSSION THREAD

**Discussion by simonw** [simonw/datasette#1013] [General]
I've started a new weekly newsletter about Datasette, hosted on Substack: https://datasette.substack.com/

I pushed out the first edition today. It covers Datasette 0.50, Git scraping, "sqlite-utils extract" and features datasette-graphql as the plugin of the week.

You can read that first newsletter online here: https://datasette.substack.com/p/datasette-050-git-scraping-extracting

**Comment by salmaaminu898-cpu** [comment]
TOOFAN V2RAY
